int16 vr_A 
int16 vr_B
int16 sw_A
int16 sw_B
int16 sw_C
int16 sw_D
int16 grp